src/ethereum/instances/instances.js

```diff
@@ -1,22 +1,16 @@
 import Web3 from 'web3';
-import {abiDispatcher, abiManager, abiEscrow, abiToken, abiWorklock} from './contract-abis'
+import {abiManager, abiEscrow, abiToken, abiWorklock} from './contract-abis'
 
-const goerliUrl = 'https://goerli.infura.io/v3/388474bfcd7f44508aefa5b952227ddb'
-const web3 = new Web3(window.ethereum || goerliUrl);
-
-// Dispatcher
-const DISPATCHER_ADDRESS = abiDispatcher[2];
-const DISPATCHER_ABI = abiDispatcher[3];
-export const instanceDispatcher = new web3.eth.Contract(DISPATCHER_ABI, DISPATCHER_ADDRESS);
+const providerUri = 'https://mainnet.infura.io/v3/d596a1f7aec940cab578b15d864781db'
+const web3 = new Web3(window.ethereum || providerUri);
 
 // Policy Manager
-const POLICY_MANAGER_ADDRESS = '0x1A0bA37398f2476F533Ae2ab622A79E310AA7680';
+const POLICY_MANAGER_ADDRESS = '0x67E4A942c067Ff25cE7705B69C318cA2Dfa54D64';  // PM Dispatcher Address
```

Comment:
It might be better to put these into a .env file
https://coderrocketfuel.com/article/how-to-load-environment-variables-from-a-.env-file-in-nodejs
But if they will never change then there's no reason to do that.

Reply:
Short of a complete hard fork and redeployment - They never change. I think it's safe to treat these proxy contract addresses as "Hardcoded" on the blockchain for eternity.